app/validators/availability_validator.rb

```diff
@@ -0,0 +1,11 @@
+class AvailabilityValidator < ActiveModel::EachValidator
```

Comment:
I like your custom validator, but there is an issue. When I try to edit the price of the existing seance I get `Start time not available` error.

Reply:
Didn't think about it. Record should be excluded. Thanks:)